File: C_Programming/count_digits.c
//Program to count number of digits in an integer.

#include <stdio.h>

int main()
{
	long long num;
	printf("Enter an integer: ");
	scanf("%lld", &num);

	int count = 0;
	while(num)
	{
		num /= 10;
		count++;
	}

	printf("\nDigits count: %d", count);

	return 0;
}

File: Python/count_digits.py
# Program to count number of digits in an integer

from math import log

number = int(input("Enter an Integer: "))
if number < 0:
	number = int(str(number)[1:])
try:
	print(int(log(number, 10))+1)
except:
	print(1)
	
